### PR TITLE
Add GitHub actions for gnitpick 

### PR DESCRIPTION
**Pull request by frimro**
Related: #27

Replaces travis with GitHub actions.

### CODE REVIEW

File: .github/workflows/gnitpick.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Set up python ${{ env.PYTHON_VERSION }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Install dependencies
+        run: pip install flake8
+
+      - name: Run gnitpick
+        run: python gnitpick.py --verbose
```
Comment (ypcs) [inline .github/workflows/gnitpick.yml]: You should use `python3` so it's explicit that we want to use that version. `python` might refer to `python2` or `python3`.

File: .github/workflows/gnitpick.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+      - name: Lint code
```
Comment (ypcs) [inline .github/workflows/gnitpick.yml]: As this is action for *running Gnitpick*, running other stuff, like linting Python code is IMHO out-of-scope for this action? This makes it much harder to use this action, as different projects might have different needs to eg. globally ignore certain errors etc. :)

Comment (frimro) [reply .github/workflows/gnitpick.yml]: Yeah, that makes sense. Probably add another action for linting the code?

File: .github/workflows/gnitpick.yml (hunk 1 above)
Comment (frimro) [reply .github/workflows/gnitpick.yml]: Updated

File: .github/workflows/gnitpick.yml (hunk 2 above)
Comment (ypcs) [reply .github/workflows/gnitpick.yml]: Yes, that would be much better :)

**Review by elguitar — APPROVED** [review]
LGTM!

**Pull request merged**